TF_Settings_Web/src/Components/Pages/Camera/Masking/MaskingScreen.tsx

```diff
@@ -23,7 +23,12 @@ export type Lens = 'Left' | 'Right';
 
 const FRAME_PROCESSING_TIMEOUT = 60;
 
-const MaskingScreen = () => {
+interface MaskingScreenProps
+{
+    tfStatus:TrackingServiceState;
```

Comment:
This doesn't appear to be used anywhere

Reply:
Yes it's unused but I left it in because it will be needed when we update the masking screen UI based on state. Can remove for now if you'd prefer.

Reply:
This is causing the build to fail. Or more specifically the unused `props` passed into the MaskingScreen